src/libraries/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -2,6 +2,9 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);netcoreapp3.1-windows;netcoreapp3.1</TargetFrameworks>
+    <IsPackable>true</IsPackable>
+    <!-- netcoreap3.0 is out of support and dropped from the package. -->
+    <NoWarn>$(NoWarn);PKV006;PKV007</NoWarn>
```

Comment:
Should the package validation not warn for out of support tfms that are dropped?

Reply:
How would that work when treating all warnings as errors?

Reply:
What do you mean? I meant that package validation ignore tfms that are out of support and not error when that tfm is dropped. So in this case where netcoreapp3.0 support is dropped from the package on the new version we are getting a package validation error, we would not get it cause the tfm being dropped is out of support.

Reply:
I misread your question. That makes total sense. Maybe controllable via a flag which defaults to true?

Reply:
Why would you need to enable validating dropping tfms that are out of support, do you see a customer scenario there? Is that something we recommend?

Reply:
Just because our support policy states that we won't support .NET Framework < 4.6 and .NETCoreApp < 3.1 when we release .NET 6, that doesn't mean that there are still thousands of customers that intentionally target lower versions. Take nunit or nlog as an example which still target .NET Framework 3.5 as they have customers that depend on it. We should not artificially lock these customers out.

https://www.nuget.org/packages/nlog/
https://www.nuget.org/packages/nunit/

Reply:
Sounds good. Then I believe we should do the framework support check by default on all tfms and have the option to disable it in out of life tfms. So make it opt-in rather than opt-out.

Reply:
I can add this feature to the package validation package. I will not block the pr on it though

Reply:
Can you open an issue to track it?

Reply:
Also, the latest package validation package, already has granular suppression support, should we consume that and suppress these granular to not loose tfm support validation?

Reply:
> Also, the latest package validation package, already has granular suppression support, should we consume that and suppress these granular to not loose tfm support validation?

I think runtime requires the enableBaselineValidation boolean flag to upgrade the version. I will try to see if i can get that change in.

Reply:
https://github.com/dotnet/sdk/issues/18583

Reply:
> I think runtime requires the enableBaselineValidation boolean flag to upgrade the version. I will try to see if i can get that change in.

Please do so. I think we should update the PackageValidation package together with this PR.